relay_test.go

```diff
@@ -54,11 +54,18 @@ func TestRelay(t *testing.T) {
 			assert.Equal(t, tt.header, string(arg2), "Header was mangled during relay.")
 			assert.Equal(t, tt.body, string(arg3), "Body was mangled during relay.")
 		}
+		stats := ts.RelayStats()
+		stats.AssertEdges(1)
+		callStats := stats.Get("client", "test", "echo")
+		require.Equal(t, 2, len(callStats), "Expected call stats for each RPC.")
+		for _, cs := range callStats {
+			cs.AssertCalled(1)
+		}
```

Comment:
Can we make this a little lighter-weight yet as thorough, something like:
```go
expected := mockstats.Expected()
expected.Get("client", "test", echo").Success()
assert.Equal(t, expected, ts.RelayStats(), "Unexpected stats")
```

If there are multiple calls, with the same call succeeding and failing, something like:
```go
expected := mockstats.Expected()
echoCall := expected.Call("client", "test", echo")
echoCall.Success().Fail("bad-request").Success()
expected.Call("client", "test", "fail").Fail("error")
assert.Equal(t, expected, ts.RelayStats(), "Unexpected stats")
```